src/status_im/i18n.cljs

```diff
@@ -52,73 +53,163 @@
+                             :lt          lt/translations
+                             :lv          lv/translations
+                             :ms          ms/translations
+                             :nb          nb/translations
+                             :ne          ne/translations
+                             :nl          nl/translations
+                             :pl          pl/translations
+                             :pt-br       pt-br/translations
+                             :pt-pt       pt-pt/translations
+                             :ro          ro/translations
+                             :ru          ru/translations
+                             :sl          sl/translations
+                             :sr          sr-rs-cyrl/translations
+                             :sr-RS_#Latn sr-rs-latn/translations
+                             :sr-RS_#Cyrl sr-rs-cyrl/translations
+                             :sv          sv/translations
+                             :sw          sw/translations
+                             :th          th/translations
+                             :tr          tr/translations
+                             :uk          uk/translations
+                             :ur          ur/translations
+                             :vi          vi/translations
+                             :zh          zh-hans/translations
+                             :zh-hans     zh-hans/translations
+                             :zh-hans-tw  zh-hans/translations
+                             :zh-hans-sg  zh-hans/translations
+                             :zh-hans-hk  zh-hans/translations
+                             :zh-hans-cn  zh-hans/translations
+                             :zh-hans-mo  zh-hans/translations
+                             :zh-hant     zh-hant/translations
+                             :zh-hant-tw  zh-hant/translations
+                             :zh-hant-sg  zh-hant/translations
+                             :zh-hant-hk  zh-hant/translations
+                             :zh-hant-cn  zh-hant/translations
+                             :zh-hant-mo  zh-hant/translations
+                             :zh-wuu      zh-wuu/translations
+                             :zh-yue      zh-yue/translations})
+
+;; english as source of truth
+(def trans-ids (set (keys en/translations)))
+
+(spec/def ::trans-id trans-ids)
+(spec/def ::trans-ids (spec/coll-of ::trans-id :kind set? :into #{}))
+
+(defn trans-ids-for-all-locales []
+  (->> translations-by-locale
+       (mapcat #(-> % val keys))
+       set))
+
+;; checkpoints
+
+(spec/def ::checkpoint.id keyword?)
+(spec/def ::checkpoint-defs (spec/map-of ::checkpoint.id ::trans-ids))
+
+(def checkpoint-1-trans-ids trans-ids)
+; these could be status versions instead of incremental numbers
+(def checkpoints-def (spec/assert ::checkpoint-defs
+                                  {::checkpoint|1 checkpoint-1-trans-ids
```

Comment:
don't use a pipe a dash would be good (if it's a variable name I'm actually confused is pipe a special syntax here ?)